src/main/java/bio/terra/service/snapshot/flight/create/SnapshotCreateFlight.java

```diff
@@ -125,10 +125,8 @@ public SnapshotCreateFlight(FlightMap inputParameters, Object applicationContext
         getDefaultRandomBackoffRetryRule(appConfig.getMaxStairwayThreads());
 
     // TODO note that with multi-dataset snapshots this will need to change
```

Comment:
I'd remove this TODO while you're doing this; we don't have a plan for multi dataset snapshots and we most likely will never have this feature.